docs/eventing/sources/pingsource.md

```diff
@@ -1,196 +1,261 @@
 ---
-title: "PingSource"
+title: "PingSource example"
 linkTitle: "PingSource"
-weight: 31
+weight: 10
 type: "docs"
+aliases:
+  - ../cronjob-source
 ---
 
-![version](https://img.shields.io/badge/API_Version-v1beta2-red?style=flat-square)
+This topic explains event-driven architecture in terms of producers and consumers and, to that end,
+shows how to configure PingSource as an event source targeting a Knative Service.
 
-A PingSource produces events with a fixed payload on a specified cron schedule.
+## Before you begin
 
-## Installation
+1. Set up [Knative Serving](../../../serving).
+1. Set up [Knative Eventing](../../../eventing).
 
-The PingSource source type is enabled by default when you install Knative Eventing.
+## Overview
```

Comment:
I really don't like overview as a heading - can we just move all the context info to above the "before you begin" instead? That way we're also not mixing steps / instructions and contextual info.

Reply:
Yep